frontend/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
```

Comment:
Is this a recommended setting from somewhere? I think the default in prettier is 2 and that's what we had in the project as well. The PR should be a lot smaller as well if we keep it at 2.

Reply:
aha, good that you noticed this! I took it from [here](https://prettier.io/docs/en/configuration.html#basic-configuration) (also saw it at other places), but if 2 is our standard, I agree we should keep that.

Reply:
fixed by 22701255906e2abeac8aa41496d8e5997f1b334f